### PR TITLE
Create Truncate and Count scripts: Alex

### LINKED ISSUE #134: Create Shell Script for Truncating Database
As a data engineer, when I am testing my database scripts I would like to be able to remove data so that I can more easily see whether my scripts populate the database.

- Create a shell script that connects to RDS and truncates the non-static tables

### PR DESCRIPTION
**Pull request by AlexOBarnes**
This closes #134 #135 

What I have done:

- Created bash script to return the count from dynamic tables
- Created bash script to truncate dynamic tables

### CODE REVIEW

File: database/truncate.sh
Comment (olliejth) [inline database/truncate.sh:1]: Very efficient, good idea

**Review by olliejth — APPROVED** [review]
Useful scripts for use. The count script is especially applicable for this stage of the project.

File: database/truncate.sh
Comment (CalebMathers) [reply database/truncate.sh:1]: I couldn't agree more

File: database/count.sh
Comment (CalebMathers) [inline database/count.sh:1]: Very useful script to quickly check if there are any increases across the dynamic tables

File: database/count.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
```
Comment (CalebMathers) [inline database/count.sh:2]: All of this without even having to enter a password, what a time to be alive

**Review by CalebMathers — APPROVED** [review]

**Review by olliejth — APPROVED** [review]
Useful ideas for shell scripts.

File: database/count.sh (hunk 1 above)
Comment (AlexOBarnes) [reply database/count.sh:2]: I know right we are really living in the future now

File: database/count.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
```
Comment (MurasatoNaoya) [inline database/count.sh:3]: Nice script for getting an understanding of what our database is holding at any given time, without having to look manually. LGTM.

File: database/truncate.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+source .env
```
Comment (MurasatoNaoya) [inline database/truncate.sh:1]: Nice way of resetting the database, in a way that can be rolled back if need be.

**Review by MurasatoNaoya — APPROVED** [review]

**Pull request merged**